M2/Macaulay2/d/hashtables.dd

```diff
@@ -318,10 +318,11 @@ lookup(object:HashTable, key:Expr):Expr; -- forward declaration
 KeyNotFound(object:string, key:Expr):Expr := (
```

Comment:
Does this work?
```suggestion
KeyNotFound(object:string, key:Expr):Expr := (
    if SuppressErrors then return nullE;
```
No reason to go through `buildErrorPacket` if we don't have to.

Reply:
That would work fine for `??` but not `try`.  we still want it to raise an error.  For example, with this change:

```m2
i1 : try ZZ.foo then "no error" else "error"

o1 = no error
```